src/Plotting.jl

```diff
@@ -506,9 +506,11 @@ function plotrxntransitorysensitivities(bsol,name,t;dSdt=nothing,tau=nothing,tol
 
     if dSdt === nothing
         if tau === nothing
-            dSdt = transitorysensitivitiesfulltrapezoidal(bsol,t)[ind,length(bsol.names)+1:end]
+            (dSdt,tau) = transitorysensitivitiesfulltrapezoidal(bsol,t)
```

Comment:
Why do we return tau here? For debugging? Is this tau used anywhere else?

Reply:
During automatic mechanism analysis we run a cluster analysis where we use tau from transitory sensitivities to select the timescale to use to separate the clusters.  I think it's also nice in general as a safety issue...I don't recall seeing any bad cases yet, but automatic tau estimation in theory isn't perfect particularly if you don't have at least one reactive-unreactive species timescale separation so it's probably good to include that in output so users can look at it without having to dig.